src/main/java/com/bina/varsim/util/DGVparser.java

```diff
@@ -146,7 +146,11 @@ public Variant parseLine() {
         VariantType type = getVariantType(ll[5].toLowerCase());
 
         // TODO right now we treat all gains as tandem duplications
-        int observedgains = (ll[15].length() > 0) ? Integer.parseInt(ll[15]) : 0;
+	if (ll[15].length() == 0)
+	    throw new RuntimeException("no observedgain");
+        int observedgains = Integer.parseInt(ll[15]);
+	if (observedgains == 0)
+	    throw new RuntimeException("observedgains = 0, not a variant, skipping");
```

Comment:
it's not really "skipping" if you throw an exception. Is this exception being caught somewhere?

Reply:
yes, see https://github.com/bioinform/varsim/pull/138/files/542467fe7d997ed140f533603dec054c13858d7f#diff-aeacb89a2526bc17574fdcc1e9e11eb0L213